reaper-menu.ini

```diff
@@ -133,92 +133,93 @@ item_25=_Ultraschall_Delete_Last_Marker Delete last marker (left to cursor)
-item_55=_Ultraschall_SkipLoopPlayback Preview cut - relative to zoom
-item_56=_Ultraschall_Mute_Selected_Items_In_Time_Range Mute selected items in time selection
-item_57=_Ultraschall_Unmute_Selected_Items_In_Time_Range Unmute selected items in time selection
-item_58=40020 Remove time selection
-item_59=-3
-item_60=-2 Chapter markers
-item_61=40326 Show chapter marker editor
-item_62=_Ultraschall_Insert_ChapterMarker Insert chapter markers at the start of every item of track
-item_63=_Ultraschall_ImportChapterMarkersFromWav Import chapter markers from WAV items
-item_64=_ULTRASCHALL_INSERT_CHAPTERS Import chapter markers from file...
-item_65=_ULTRASCHALL_SAVE_CHAPTERS Export chapter markers to file...
-item_66=_ULTRASCHALL_SAVE_CHAPTERS_TO_PROJECT Export chapter markers to project folder
-item_67=-3
-item_68=-1
-item_69=-2 Export
-item_70=_Ultraschall_Export_Assistant Start Export Assistant
-item_71=_Ultraschall_Edit_ID3_Tags Edit ID3V2 metadata
-item_72=_ULTRASCHALL_INSERT_MEDIA_PROPERTIES Insert metadata into MP3 target...
-item_73=_Ultraschall_ResetPlaybackRate_and_RenderProject Render project...
-item_74=-3
-item_75=_Ultraschall_Open_Project_Folder Open project folder
-item_76=-1
-item_77=-4 Miscellaneous
-item_78=-1
-item_79=-2 View
-item_80=40251 Show routing matrix
-item_81=_Ultraschall_Snapshot_Editor Show routing snapshots
-item_82=40422 Show screen/track/item sets window
-item_83=_Ultraschall_Toggle_Follow Toggle follow mode
-item_84=40651 Toggle item labels
-item_85=_S&M_CYCLACTION_10 Toggle visibility of mute envelopes
-item_86=40891 Toggle mute envelope in lane for selected track
-item_87=_Ultraschall_Toggle_Peak_Gain_Normal_Max Toggle peak gain: normal - mid - max
-item_88=42307 Toggle rectify peaks
-item_89=_Ultraschall_Toggle_Arrange_Zoom Toggle arrange zoom
-item_90=-3
-item_91=-2 Navigation
-item_92=40173 Go to next marker
-item_93=40172 Go to previous marker
-item_94=40319 Go to next item
-item_95=40318 Go to previous item
-item_96=_Ultraschall_FocusPosition Go to play position
-item_97=40630 Go to start of time selection
-item_98=40631 Go to end of time selection
-item_99=_Ultraschall_Safemode_Start_Pause Start/Pause
-item_100=_Ultraschall_Safemode_Start_Stop Start/Stop
-item_101=-3
-item_102=-2 Customize
-item_103=40605 Shortcuts and action list...
-item_104=40899 Set track icon...
-item_105=41930 Show theme configuration window...
-item_106=40690 Show theme element finder...
-item_107=42074 Peaks Display Settings
-item_108=_Ultraschall_Set_Colors_To_Sonic_Rainboom Set Colors to Gentle Sonic Rainboom
-item_109=_Ultraschall_Set_Colors_To_Sonic_Rainboom_Spread Set Colors to Adjusted Sonic Rainboom
-item_110=_SWSAUTOCOLOR_OPEN Auto color...
-item_111=-3
-item_112=_Ultraschall_KeyMap Ultraschall Keymap (url)
-item_113=-1
-item_114=_Ultraschall_StartScreen Show Startscreen / About Ultraschall...
+item_29=_RS3e02d5f336283a81d2cf480f1cfe234b7fa9ff6a Set marker right from cursor to planning stage
```

Comment:
Das müsste man wohl noch umbauen damit @mespotine nicht meckert :-)

Reply:
^^

Reply:
Lustig ans Werk :-)

Reply:
Holy shit.
Kann man die kb.ini irgendwie modularisieren. Das ist ja super unübersichtlich.
Z. B. das eigene Podcast-Menü als eigene ini Datei, die dann hier eingebunden wird?

Reply:
Nein, kann man nicht. Bei sowas reagiert REAPER eher zickig. Bedenke: zum editieren der Menüs geht man nicht über diese .ini sondern hat dazu in REAPER selbst eine brauchbare GUI:

<img width="1129" alt="image" src="https://user-images.githubusercontent.com/3195116/36939894-1dd49446-1f39-11e8-9459-69b453503e75.png">